packages/argo-admin/src/extension-api/index.ts

```diff
@@ -1,19 +1,6 @@
-export type {LayoutApi} from './layout';
-export {isLayoutApi} from './layout';
-
-export type {LocaleApi} from './locale';
-export {isLocaleApi} from './locale';
-
-export type {ContainerApi, ExtensionContainer} from './container';
-
-export {isContainerApi} from './container';
-
-export type {DataApi, ExtensionData} from './data';
-
-export {isDataApi} from './data';
-
-export type {SessionTokenApi} from './sessionToken';
-export {isSessionTokenApi} from './sessionToken';
-
-export type {ToastApi} from './toast';
-export {isToastApi} from './toast';
+export * from './layout';
+export * from './locale';
+export * from './container';
+export * from './data';
+export * from './sessionToken';
+export * from './toast';
```

Comment:
I think this causes problems for the tsdoc generator. cc: @iainmcampbell

Reply:
yeah, it will break tsdoc. would be good if we can keep this file as it was previously. did `tsc` do this reformatting automatically? I had problems with `tsc` reformatting my `components/index.ts` file like this too, very annoying

Reply:
No, this was unrelated to auto-formatting. The change was made because we are missing some required exports argo-admin-react was relying on when nested subfolder import is no longer supported.

@iainmcampbell can you walk me through what tsdoc wants? Is this format with multiple type exports ok?

```
export type {LayoutApi, Layout} from './layout';
export {isLayoutApi} from './layout';
````

Reply:
> Is this format with multiple type exports ok?

yeah, the only thing tsdoc doesn't like is `export *`. if your exports are all named (`{LayoutApi, Layout}`) you're good 👌